conf/config.toml

```diff
@@ -116,6 +116,10 @@
 ## When PD fails to receive the heartbeat from a store after the specified period of time,
 ## it adds replicas at other nodes.
 # max-store-down-time = "30m"
+## Controls the time interval between write hot regions info into leveldb
+# hot-regions-write-interval= "20m"
```

Comment:
I suggest 10m, and the default `hot-regions-reserved-days` 7 days may be enough. and could you add a comment about How much storage size will use with the mock?

Reply:
If the write interval is 10 minutes; If only data is written, the data size of a month is about 550M. Both data is written and deleted in a month. The data size of a month is about 880M

Reply:
If the write interval is 10 minutes; If only data is written, the data size of seven days is about 141M. Both data is written and deleted in a month. The data size of a month is about 220M